### ISSUE TITLE
[Bug] Investigate scheduled triggers for nightly scan 

### ISSUE THREAD

**Issue by ADPennington** [raft-tech/TANF-app#2403]
**Description:**

Since v3.0.0 was merged into HHS/TANF-app main and master branches, the [scheduled trigger](https://github.com/raft-tech/TANF-app/pull/2284) for the nightly OWASP scan is triggering all the workflows instead of just the nightly scan (see below). 

The [recent history and insights from CircleCI for raft-tech/TANF-app](https://app.circleci.com/insights/github/raft-tech/TANF-app/workflows/build-and-test/overview?branch=develop&reporting-window=last-90-days) suggest that all workflows (except those recently implemented --eg. `enable-versioning-for-s3-buckets`) have been running daily for at least the last 90 days. 

Until this is resolved, the scheduled triggers intended for the nightly scan on the HHS side have been disabled. 

_develop (raft-tech)_
<img width="1189" alt="Screenshot 2023-02-08 at 8 04 54 PM" src="https://user-images.githubusercontent.com/63075587/217688792-24f1e4ad-2c50-4e2c-b05e-1eb5806f7ad6.png">

_main and master (HHS)_

<img width="1175" alt="Screenshot 2023-02-08 at 8 11 23 PM" src="https://user-images.githubusercontent.com/63075587/217689293-96dbc987-05ba-4bcd-a82e-483db7875b79.png">

**Acceptance Criteria:** 
- [ ] Nightly workflow triggers at 12AM UTC, daily
- [ ] no other workflows trigger at 12AM UTC, daily
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
- [ ] _troubleshoot_
- [ ] _update circleci config_
- [ ] Run Testing Checklist and confirm all tests pass

**Supporting Documentation:**
- [about scheduled workflows with dynamic config](https://support.circleci.com/hc/en-us/articles/360060833331-Support-for-Scheduled-Workflows-in-Dynamic-Configurations-Setup-Workflows)
- [workaround ](https://discuss.circleci.com/t/workaround-using-scheduled-workflows-with-dynamic-config/40344)

**Comment by ADPennington** [comment]
since #2414 was merged, we've not seen a successful nightly scan. its appropriately triggering the nightly scan workflow only, but the run is not successful. see [here](https://app.circleci.com/insights/github/raft-tech/TANF-app/workflows/nightly/overview?branch=develop)

**Comment by ADPennington** [comment]
I've been keeping an eye on the scheduled trigger this week across the 3 environments, and it appears to me that only the nightly workflow is triggering since this ticket was addressed, which is great. 

Still, it's odd that the workflow is intermittently failing at the same step (front end nightly scan) across all env and it's unclear why. 

I know that @George-Hudson had a circle ci ticket about this and I submitted another one today, 3/3, to flag that this is happening across branches and repos. 

it seems to me that this is a new issue and that a new ticket can be created to track it instead of using this one but open to your suggestions @andrew-jameson @Smithh-Co @George-Hudson 🙂
